python/HeaderService/hslib.py

```diff
@@ -319,7 +319,8 @@ def announce(self):
         kw = {'byteSize': bytesize,
               'checkSum': md5value,
               'generator': 'ATHeaderService',
-              'mimeType': 'FITS',
+              # Make sure this is also changed on hslib_salobj when merged
+              'mimeType': self.write_mode.upper(),
```

Comment:
Is this meant to be a real mime type or are you using mime type here to be an internal format name?

Reply:
For now this is only and internal format name.

Reply:
I'd call it `fileType` or something then. MIME type implies something that you are not giving.

Reply:
That is the definition for a LFO (Large File Object) message from T&S, not mine. If you prefer we could use a proper mimeType like `text/yaml` for yaml. I kind of prefer this last option, but maybe Archiver doesn't like the extra complexity.

Reply:
Eek. Calling something a MIME type that is not a MIME type seems odd to me. YAML is generally `application/vnd.yaml` but a lot of people use `text/yaml`. FITS is `application/fits`.

Reply:
I like this. I'll write a little function to assign `application/fits` to FITS mode and `text/yaml` for YAML mode.